tests/RobotFramework/tests/cumulocity/configuration/configuration_operation.robot

```diff
@@ -134,6 +158,34 @@ Manual config_update operation request
     ...    expected_status=failed
     ...    c8y_fragment=c8y_DownloadConfigFile
 
+Trigger config_update operation from another workflow
+    Set Device Context    ${PARENT_SN}
+
+    Execute Command     curl -X PUT --data-binary 'new content for CONFIG1' "http://${PARENT_IP}:8000/tedge/file-transfer/${PARENT_SN}/sub_config_update/sub-2222"
+    Publish and Verify Local Command
+    ...    topic=te/device/main///cmd/sub_config_update/sub-2222
+    ...    payload={"status":"init","tedgeUrl":"http://${PARENT_IP}:8000/tedge/file-transfer/${PARENT_SN}/sub_config_update/sub-2222","remoteUrl":"","type":"CONFIG1"}
+    ...    expected_status=successful
+    ...    c8y_fragment=c8y_UploadConfigFile
```

Comment:
```suggestion
    ...    c8y_fragment=c8y_DownloadConfigFile
```
Since the test is validating a negation, it was passing, but I'm assuming that you really meant this.

Reply:
Fixed by https://github.com/thin-edge/thin-edge.io/pull/2844/commits/ca0f4cde902c638786c6f57a25b7df6781030a1f